bin/mocha

```diff
@@ -16,7 +16,7 @@ const args = [path.join(__dirname, '_mocha')];
 // Must be loaded here to handle node-specific options
 getOptions();
 
-process.argv.slice(2).forEach(arg => {
+process.execArgv.forEach(arg => {
```

Comment:
This breaks a lot of stuff... I'm not sure this is what you meant to do.  Basically any argument that isn't a `node` argument won't get passed to `_mocha`.

Reply:
(see [the build](https://travis-ci.org/mochajs/mocha/builds/340422371?utm_source=github_status&utm_medium=notification) for what's going on)